rllib/BUILD

```diff
@@ -1618,7 +1618,7 @@ py_test(
 py_test(
     name = "connectors/tests/test_agent",
     tags = ["team:rllib", "connector"],
-    size = "small",
+    size = "medium",
```

Comment:
this is from some other pr right?

Reply:
The other PR will get merged and this difference will go away. I wanted to make sure the tests on CI doesn't get red b/c of time outs.